Data_Exploration.R

```diff
@@ -0,0 +1,34 @@
+#let's load some packages we may need
+library(dplyr)
+library(ggplot2)
+
+#let's load our dataset
+data(midwest)
+
+#let's find the metro areas with the greatest population density of american indians in the midwest
+midwest %>%
+  select(PID, county, state, area, poptotal, popdensity, popamerindian, percamerindan, inmetro) %>%
+  filter(inmetro == 1) %>%
+  mutate("amerindian_pd" = popamerindian / area) %>%
+  arrange(desc(amerindian_pd))
+  
+#let's find the non-metro areas with the greatest population density of american indians in the midwest
+midwest %>%
+  select(PID, county, state, area, poptotal, popdensity, popamerindian, percamerindan, inmetro) %>%
+  filter(inmetro == 0) %>%
+  mutate("amerindian_pd" = popamerindian / area) %>%
+  arrange(desc(amerindian_pd))
+
+#let's find the state with the most non-metro counties with a population density
+#of american indians over 4000.
+midwest %>%
+  select(PID, county, state, area, poptotal, popdensity, popamerindian, percamerindan, inmetro) %>%
+  group_by(state) %>%
+  filter(inmetro == 0) %>%
+  mutate("amerindian_pd" = popamerindian / area) %>%
+  filter(amerindian_pd > 4000) %>%
+  arrange(desc(amerindian_pd)) %>%
+  count(state)
+
```

Comment:
Your code is easy to follow and is organized nicely. I didn't have any trouble understanding what you were trying to do as you explored the data. You also did a good job of using the different dplyr functions.

My only issue was that I wasn't sure where the midwest dataset that you used was located, so I didn't get the chance to reproduce your work. It would've also been nice to see what some of this data looked like visually.